static/js/components/tiles/top_event_tile.tsx

```diff
@@ -132,7 +132,13 @@ export const TopEventTile = memo(function TopEventTile(
                 <tr>
                   <td></td>
                   {showNameColumn && <td>Name</td>}
-                  {showPlaceColumn && <td>{props.enclosedPlaceType}</td>}
+                  {showPlaceColumn && (
+                    <td>
+                      {Object.keys(eventPlaces).length < topEvents.length
```

Comment:
For my own understanding, why compare number of event places with number of top events?

Reply:
eventPlaces has a key value pair for every event that has a place found for it. Therefore, if there are less eventPlaces entries than number of events, that means at least one event is missing the place